README.md

```diff
@@ -897,6 +898,45 @@ stage('Gitflow') {
 * `gitflow.finishRelease(releaseVersion)` - Finishes a git release by merging into develop and master.
    * Use the `releaseVersion` (String) as the name of the new git release. 
    
+# SCMManager
+
+Provides the functionality to handle pull requests on a SCMManager repository.
+
+You need to pass `usernamePassword` (i.e. a String containing the ID that refers to the
+[Jenkins credentials](https://jenkins.io/doc/book/using/using-credentials/)) to `SCMManager` during construction.
+These are then used for handling the pull requests.
+
+```groovy
+SCMManager scmm = new SCMManager(this, 'ourCredentials')
+
+```
+
+Set the repository url through the `repositoryUrl` property like so:
+
+```groovy
+SCMManager scmm = new SCMManager(this, 'ourCredentials')
+scmm.repositoryUrl = "hostname/scm/api/v2/pull-requests/backend/myrepo"
```

Comment:
Passing the specific PullRequest URL as `repositoryUrl` is bound to cause problems.
The name implies a general reop URL, but the content is expected as specific pull request URL.
What if the SCMManager class ever gets features other than pull requests?
What if the URL ever changes?
I think it would be easier to maintain and understand if we passed the hostname and repo as different parameters.

Reply:
Also, we should pass the whole url, including protocol.
This way it also allows for testing with `http://` only.

Reply:
Fixed in 2d6a8c4b